hacspec-scrambledb/scrambledb/src/table.rs

```diff
@@ -20,21 +20,53 @@ pub type BlindTable = MultiColumnTable<BlindIdentifier, EncryptedValue>;
 pub type ConvertedTable = SingleColumnTable<BlindPseudonym, EncryptedValue>;
 pub type PseudonymizedTable = SingleColumnTable<Pseudonym, PlainValue>;
 
-#[derive(Clone)]
-pub struct Column<K, V> {
+use std::fmt::Debug;
```

Comment:
This file really needs to be cleaned up. But not in this PR.

Reply:
Fully agree!